java/src/jmri/jmrit/operations/trains/TrainsTableModel.java

```diff
@@ -522,7 +522,12 @@ public synchronized void propertyChange(PropertyChangeEvent e) {
                 log.debug("Update train table row: {} name: {}", row, train.getName());
             }
             if (row >= 0) {
-                _table.scrollRectToVisible(_table.getCellRect(row, 0, true));
```

Comment:
Wrap this in ```SwingUtilities.invokeLater()``` [JavaDoc](http://docs.oracle.com/javase/8/docs/api/javax/swing/SwingUtilities.html#invokeLater-java.lang.Runnable-)